src/CSharpFunctionalExtensions.FluentAssertions/ResultExtensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿using FluentAssertions;
+using FluentAssertions.Execution;
+using FluentAssertions.Primitives;
+
+namespace CSharpFunctionalExtensions.FluentAssertions;
+
+public static class ResultExtensions
+{
+    public static ResultAssertions Should(this Result instance) => new(instance);
+}
+
+public class ResultAssertions : ReferenceTypeAssertions<Result, ResultAssertions>
+{
+    public ResultAssertions(Result instance) : base(instance) { }
+
+    protected override string Identifier => "Result";
+
+    /// <summary>
+    /// 
```

Comment:
It's definitely worth going through and filling out these XML doc comments, even with something simple.

If I use a method from a library and it doesn't have an XML comment, I pause and question if it's an internal API that I shouldn't be using.

Reply:
I let copilot generate some comments for me. 😄  We can update them as needed.